asciidoctorj-core/src/main/java/org/asciidoctor/Attributes.java

```diff
@@ -332,7 +333,11 @@ public void setTableOfContents(Placement placement) {
      *            value.
      */
     public void setShowTitle(boolean showTitle) {
-        this.attributes.put(SHOW_TITLE, showTitle);
+        if (showTitle) {
+            this.attributes.put(SHOW_TITLE, true);
+        } else {
+            this.attributes.put(NOTITLE, true);
```

Comment:
Maybe I have to remove the other attribute, so that the logic stays consistent if somebody calls `.setShowTitle(true).setShowTitle(false)`.